### PR TITLE
Update it.coffee

### PR DESCRIPTION
**Pull request by TAFKAR**
Tried to add some translation excluded strings. Hope it's good. Hope it helps with #105.

### CODE REVIEW

File: t9n/it.coffee [hunk 1]
```diff
@@ -103,6 +107,9 @@ it =
       "User not found": "Utente non trovato"
       "Verify email link expired": "Link per la verifica dell'email scaduto"
       "Verify email link is for unknown address": "Il link per la verifica dell'email fa riferimento ad un indirizzo sconosciuto"
+      "At least 1 digit, 1 lowercase and 1 uppercase": "Almeno un numero, 1 carattere minuscolo e 1 maiuscolo"
```
Comment (splendido) [inline t9n/it.coffee]: This might be too specific, since "1"s could be easily changed from app to app...

Comment (TAFKAR) [inline t9n/it.coffee]: I've tried to find the specific variables that make the period in code but instead I found the exact period itself.

**Comment by softwarerero** [comment]
You could do:
```
"At least 1 digit, 1 lowercase and 1 uppercase": "Almeno @{digit} numero, @{lowercase} carattere minuscolo e @{uppercase} maiuscolo"
...
T9n.get 'At least 1 digit, 1 lowercase and 1 uppercase', true, {digit: 'un', lowercase: 1, uppercase: 1}
```

**Comment by gVolop** [comment]
what about the message that is appear on first login through the verified email link ? 

screen shot 2015-12-23 at 2 11 28 pm

how can i make it with translate?

**Pull request closed without merging**